build.gradle

```diff
@@ -22,12 +27,20 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jdbc'
 	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
 	
-	compile('org.springframework.boot:spring-boot-starter')
-	compile 'org.springframework.boot:spring-boot-starter-web'
-	compile 'org.springframework.boot:spring-boot-starter-data-jpa'
-	compile 'com.fasterxml.jackson.core:jackson-core'
-	compile 'com.fasterxml.jackson.core:jackson-annotations'
+	compile("org.springframework.boot:spring-boot-starter")
+	compile("org.springframework.boot:spring-boot-starter-web")
+	compile("org.springframework.boot:spring-boot-starter-data-jpa")
+	compile("com.fasterxml.jackson.core:jackson-core")
+	compile("com.fasterxml.jackson.core:jackson-annotations")
 	compile("com.fasterxml.jackson.core:jackson-databind")
+	compile("org.springframework.kafka:spring-kafka")
+	compile("org.apache.kafka:kafka-streams")
+	compile("org.springframework.cloud:spring-cloud-stream")
+	compile("org.springframework.cloud:spring-cloud-stream-test-support")
+	compile("org.springframework.cloud:spring-cloud-stream-binder-kafka")
+	compile("org.springframework.cloud:spring-cloud-stream-binder-kafka-streams")
+	compile ("com.vladmihalcea:hibernate-types-52:2.9.11")
```

Comment:
do we still use this?  I thought we removed the need for native postgres arrays.

Reply:
Removed.